python_ta/reporters/templates/template.html

```diff
@@ -0,0 +1,100 @@
+<html>
+   <head>
+      <style type="text/css">
+         {% include "stylesheet.css" %}
+      </style>
+   </head>
+   <div id="preheader">
+      <div id="module">
+         Module analysed: {{ mod_name }}
+      </div>
+      <span id="date">
+      {{ date_time }}
+      </span>
+   </div>
+   <div id="wrap">
+      <div class="title">
+         PyTA Error Report
+      </div>
+   </div>
+   <div id="inside">
+      {% for file in messages_by_file %}
+      <div id="core">
+         <h3> {{ file.filename }} </h3>
+         <div id="code">
+            {{ code_err_title }}
+         </div>
+         {% for message in file.code %}
+         <div class="slider">
+            <p style="font-size: 110%">
+               <span class="code_id">
+               <a href="http://www.cs.toronto.edu/~david/pyta/&#35;{{message}}" target="_blank">{{ message }} ({{ file.code[message][0].symbol }})</a>
+               </span>
+               &nbsp;&nbsp;Number of occurrences: {{ file.code[message]|length }}
+               <button class="mdl-button mdl-js-button mdl-button--raised"> Expand/Collapse </button>
```

Comment:
either remove unused classes, or add the functionality that the classes are for.